plugins/inputs/jolokia/jolokia.go

```diff
@@ -14,7 +14,16 @@ import (
 	"github.com/influxdata/telegraf/plugins/inputs"
 )
 
-type Server struct {
+type JolokiaServer struct {
+	Name     string
+	Host     string
+	Username string
+	Password string
+	Port     string
+	Https    bool
+}
```

Comment:
+1 to @phemmer, it would be preferable to just have the jolokia plugin accept a url